packages/manager/modules/advices/rollup.config.js

```diff
@@ -0,0 +1,22 @@
+import rollupConfig from '@ovh-ux/component-rollup-config';
+
+const config = rollupConfig({
+  input: 'src/index.js',
+});
+
+const outputs = [config.es()];
+
+if (process.env.BUILD === 'production') {
+  outputs.push(config.cjs());
+  outputs.push(
+    config.umd({
+      output: {
+        globals: {
+          angular: 'angular',
+        },
+      },
+    }),
+  );
+}
```

Comment:
```suggestion
```
as this package is private, we don't publish it on a registry therefore only monorepo package can use it and we only need `esm` build

Reply:
👍